app/models/user.rb

```diff
@@ -3,4 +3,11 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable, :confirmable
+
+  has_many :recipes, foreign_key: 'user_id'
+  has_many :foods, foreign_key: 'user_id'
```

Comment:
Hi @karayamanemre,
You have done a great job in implementing the associations for the user. However, you did not implement cascading deletes for the associations. Not doing so will prevent the user from being deleted if need be. Kindly add `dependent: destroy` to the association to allow the deletion of a user and its associated models.